lte/cloud/go/services/subscriberdb/digest.go

```diff
@@ -107,27 +107,29 @@ func GetPerSubscriberDigestsDiff(prev []*lte_protos.SubscriberDigestWithID, next
 	deleted := []string{}
 
 	for i < n && j < m {
-		iSid, jSid := prev[i].Sid.Id, next[j].Sid.Id
-		if iSid == jSid {
+		iSidStr, jSidStr := lte_protos.SidString(prev[i].Sid), lte_protos.SidString(next[j].Sid)
```

Comment:
Why the change?

Reply:
Through this change, the entire string of `Sid` will be sent down to agw and stored in the agw database, instead of only the `Sid.Id` (e.g. "IMSI00000" vs. "00000") @hcgatewood

Reply:
Gotcha. Didn't realize that's how it worked. So I like the content of the change, but just keep the var names as `iSid` and `jSid` for conciseness